base/tools/src/main/java/com/netscape/cmstools/cli/MainCLI.java

```diff
@@ -578,6 +579,14 @@ public void init() throws Exception {
                     password.clear();
                 }
             }
+            manager.setPasswordCallback(new NullPasswordCallback());
```

Comment:
I suppose this must be done after all `login()` calls are done? I'm not sure if we ever need it, but basically subsequent `login()` calls probably won't work.

Or maybe should we use this instead?
https://github.com/dogtagpki/jss/blob/master/base/src/test/java/org/mozilla/jss/tests/FilePasswordCallback.java

Reply:
This will set the default login. The above tokens with explicit login should not be modified by this operation and should not change their behaviour. The `FilePasswordCallback` could be a good solution if used as default when a file is provided but it requires a change in JSS and some reorganisation in MainCLI. We can do it in future update.

For future login it should not be a problem for the CLI. If the command should fails the user has to update the password file and execute again since there is no console in this case. It would be different for server side operation but it is not this case.